prow/plugins/approve/approve.go

```diff
@@ -41,6 +41,7 @@ const (
 	PluginName = "approve"
 
 	approveCommand       = "APPROVE"
+	wrongApproveCommand  = "APPROVED"
```

Comment:
nit: `wrongApproveCommand` implies error to me, how about change it to `misspelledApproveCommand`? 

alternatively, it might also take a slice like `alternativeApproveCommands`, or just combine with `APPROVE` to be underl `approveCommands`

Reply:
ok, I went with `alternateApproveCommand`. Doing a slice would have required a bunch more rewriting...